plugins/discovery-azure-classic/build.gradle

```diff
@@ -125,107 +125,122 @@ thirdPartyAudit.excludes = [
+    'javax.xml.bind.annotation.XmlAccessorType',
+    'javax.xml.bind.annotation.XmlAnyAttribute',
+    'javax.xml.bind.annotation.XmlAnyElement',
+    'javax.xml.bind.annotation.XmlAttachmentRef',
+    'javax.xml.bind.annotation.XmlAttribute',
+    'javax.xml.bind.annotation.XmlElement$DEFAULT',
+    'javax.xml.bind.annotation.XmlElement',
+    'javax.xml.bind.annotation.XmlElementDecl$GLOBAL',
+    'javax.xml.bind.annotation.XmlElementDecl',
+    'javax.xml.bind.annotation.XmlElementRef$DEFAULT',
+    'javax.xml.bind.annotation.XmlElementRef',
+    'javax.xml.bind.annotation.XmlElementRefs',
+    'javax.xml.bind.annotation.XmlElementWrapper',
+    'javax.xml.bind.annotation.XmlElements',
+    'javax.xml.bind.annotation.XmlEnum',
+    'javax.xml.bind.annotation.XmlEnumValue',
+    'javax.xml.bind.annotation.XmlID',
+    'javax.xml.bind.annotation.XmlIDREF',
+    'javax.xml.bind.annotation.XmlInlineBinaryData',
+    'javax.xml.bind.annotation.XmlList',
+    'javax.xml.bind.annotation.XmlMimeType',
+    'javax.xml.bind.annotation.XmlMixed',
+    'javax.xml.bind.annotation.XmlNs',
+    'javax.xml.bind.annotation.XmlNsForm',
+    'javax.xml.bind.annotation.XmlRegistry',
+    'javax.xml.bind.annotation.XmlRootElement',
+    'javax.xml.bind.annotation.XmlSchema',
+    'javax.xml.bind.annotation.XmlSchemaType$DEFAULT',
+    'javax.xml.bind.annotation.XmlSchemaType',
+    'javax.xml.bind.annotation.XmlSchemaTypes',
+    'javax.xml.bind.annotation.XmlSeeAlso',
+    'javax.xml.bind.annotation.XmlTransient',
+    'javax.xml.bind.annotation.XmlType$DEFAULT',
+    'javax.xml.bind.annotation.XmlType',
+    'javax.xml.bind.annotation.XmlValue',
+    'javax.xml.bind.annotation.adapters.CollapsedStringAdapter',
+    'javax.xml.bind.annotation.adapters.HexBinaryAdapter',
+    'javax.xml.bind.annotation.adapters.NormalizedStringAdapter',
+    'javax.xml.bind.annotation.adapters.XmlAdapter',
+    'javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter$DEFAULT',
+    'javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter',
+    'javax.xml.bind.annotation.adapters.XmlJavaTypeAdapters',
+    'javax.xml.bind.attachment.AttachmentMarshaller',
+    'javax.xml.bind.attachment.AttachmentUnmarshaller',
+    'javax.xml.bind.helpers.AbstractMarshallerImpl',
+    'javax.xml.bind.helpers.AbstractUnmarshallerImpl',
+    'javax.xml.bind.helpers.DefaultValidationEventHandler',
+    'javax.xml.bind.helpers.Messages',
+    'javax.xml.bind.helpers.NotIdentifiableEventImpl',
+    'javax.xml.bind.helpers.ParseConversionEventImpl',
+    'javax.xml.bind.helpers.PrintConversionEventImpl',
+    'javax.xml.bind.helpers.ValidationEventImpl',
+    'javax.xml.bind.helpers.ValidationEventLocatorImpl',
+    'javax.xml.bind.util.JAXBResult',
+    'javax.xml.bind.util.JAXBSource$1',
+    'javax.xml.bind.util.JAXBSource',
+    'javax.xml.bind.util.Messages',
+    'javax.xml.bind.util.ValidationEventCollector'
+  ]
+} else {
+  thirdPartyAudit.excludes += [
```

Comment:
Might be worth a comment that these were added to the jre in 9 but we still need to include the jar because we expect to run in 8.